pkg/kubelet/kubelet_node_status.go

```diff
@@ -56,6 +56,9 @@ const (
 	// maxNamesPerImageInNodeStatus is max number of names per image stored in
 	// the node status.
 	maxNamesPerImageInNodeStatus = 5
+
+	// mb is used to easily convert an int to an mb
+	mb = 1024 * 1024
```

Comment:
It was used in a test file.  But the numbers didnt actually matter, so I removed this.